lib/controllers/skill-simulation-controller/index.js

```diff
@@ -14,13 +15,14 @@ module.exports = class SkillSimulationController {
         if (configuration === undefined) {
             throw 'Cannot have an undefined configuration.';
         }
-        const { skillId, locale, stage, profile, debug, smapiClient } = configuration;
+        const { skillId, locale, stage, profile, output, debug, smapiClient } = configuration;
         this.profile = profile;
         this.doDebug = debug;
         this.smapiClient = smapiClient || new SmapiClient({ profile: this.profile, doDebug: this.doDebug });
         this.skillId = skillId;
         this.locale = locale;
         this.stage = stage;
+        this.dialogOutputInstance = new DialogOutputFile(output);
```

Comment:
using the `skill-io` idea, can we change all the output word to `SkillIO`? for this case, it's `skillIOInstance`